app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less

```diff
@@ -36,6 +36,17 @@
             min-height: 50px;
         }
     }
+
+    .page-actions {
+        > button {
+            float: right;
+        }
+        > button#back {
+            float: left;
```

Comment:
Hi @amol2jcommerce. Thanks for collaboration. Can you please explain why do you use `float` with `absolute` together?

Reply:
Hi @VladimirZaets 

This issues occurring before page load, after applying float: right this will not look proper as pet expected result. please check the screen.

![1](https://user-images.githubusercontent.com/18118631/51161750-ace1b400-18b9-11e9-8894-c9c41d70499c.jpg)

After page load this will look proper

![2](https://user-images.githubusercontent.com/18118631/51161795-dc90bc00-18b9-11e9-9c97-5acd4ef3edb5.jpg)

After page load html structure will change that's why result is proper.
Therefore I have applied position: absolute property to set the back button at the right place.